tests/spec/code_series_ops/test_containment.py

```diff
@@ -51,3 +64,27 @@ def test_is_in_codelist_csv(spec_test):
             4: None,
         },
     )
+
+
+def test_is_not_in_codelist_csv(spec_test):
```

Comment:
I don't actually think we need this test. I think the first one you added does enough to specify the behaviour of `is_not_in` with respect to `is_in` and everything else follows from that.

Reply:
thx!